app/src/main/java/comcmput301f16t01/github/carrier/Requests/RequestController.java

```diff
@@ -151,12 +152,19 @@ public void searchByKeyword(String keyword) {
         }
     }
 
-        /**
+    /**
      * Search requests by a location. This sets it so the singleton contains the information for
      * this query. Use getResults() to get the information.
      */
-    public void searchByLocation( /* location parameters? */ ) {
-
+    public void searchByLocation(Location location) {
+        // TODO check how these are sorted, we want to sort them by those closest to those furthest away
```

Comment:
Did you complete this TODO while working on this branch? (I remember you mentioning it)